### PR TITLE
adjusted backend and backend testing

### PR DESCRIPTION
**Pull request by pabucsb**
In this PR I started EPIC: Adjust Ride Request fields to clarify pickup and dropoff. I  renamed the variable room within the entity file in order to avoid confusion. Added new member variables pickupRoom and notes. Adjusted the controller to reflect these changes and also adjusted controller tests. I included a screenshot of a swagger endpoint to show that the changes were successful.

image

### CODE REVIEW

**Review by alecmorrison — DISMISSED** [review]
Looks good!

File: src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java [hunk 1]
```diff
@@ -90,25 +90,35 @@ public Ride postRide(
...
+        @Parameter(name="notes", description="String, Notes for additional information", example="Library entrance by the arbor", required = false)
+        @RequestParam String notes
+
+
         )
         {
 
         Ride ride = new Ride();
-        
+        //not sure if we have to set dropoffroom and pickuproom
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java:110]: yes, you should, and you did.  So we can remove this comment.
```suggestion
```

**Review by pconrad — CHANGES_REQUESTED** [review]
One comment to remove, otherwise LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**